### PR TITLE
test: add firehose test

### PR DESCRIPTION
**Pull request by giovtorres**
This change adds a test case for serializing and deserializing a Firehose formatted event message.

### CODE REVIEW

File: src/test/java/com/mcneilio/FirehoseTest.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+import com.mcneilio.shokuyoku.format.Firehose;
+import org.json.JSONObject;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.contrib.java.lang.system.EnvironmentVariables;
+
+public class FirehoseTest {
+
+    JSONObject testMessage = new JSONObject();
```
Comment (mcneiljt) [inline src/test/java/com/mcneilio/FirehoseTest.java:13]: Seems like this should be scoped to the test method?

File: src/test/java/com/mcneilio/FirehoseTest.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+    @Rule
+    public final EnvironmentVariables environmentVariables = new EnvironmentVariables();
```
Comment (mcneiljt) [inline src/test/java/com/mcneilio/FirehoseTest.java:16]: Does this need to be public and scoped to the class?

**Comment by mcneiljt** [comment]
Looks like this PR includes the files from the previous PR.

**Comment by giovtorres** [comment]
> Looks like this PR includes the files from the previous PR.

Yeah, just had to rebase this one.

File: src/test/java/com/mcneilio/FirehoseTest.java (hunk 2 above)
Comment (giovtorres) [reply src/test/java/com/mcneilio/FirehoseTest.java:14]: I believe so, since it is a junit rule.

**Review by mcneiljt — APPROVED** [review]

**Pull request merged**